apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -178,17 +184,26 @@ export class TldrawApp {
 		const nextRecentFileOrdering = []
 
 		for (const fileId of myFileIds) {
+			const file = myFiles[fileId]
+			const state = myStates[fileId]
 			const existing = this.lastRecentFileOrdering?.find((f) => f.fileId === fileId)
+			// File was not loaded before, but it is now
+			if (file && existing && !existing.isLoaded) {
+				nextRecentFileOrdering.push({
+					...existing,
+					isLoaded: true,
+				})
+				continue
+			}
 			if (existing) {
 				nextRecentFileOrdering.push(existing)
 				continue
 			}
-			const file = myFiles[fileId]
-			const state = myStates[fileId]
 
```

Comment:
I think the behavior of this PR can be achieved with a one-liner

```suggestion
if (!file) continue
```

Reply:
D'oh! 🤦